### PR TITLE
Add comment section

### PR DESCRIPTION
**Pull request by Fsher07**
-Send comments to api
-Get comments from api
- Display all comments on related windows

### CODE REVIEW

File: modules/add.js [hunk 1]
```diff
@@ -55,11 +55,31 @@ export const popupComments = async (meal) => {
...
+  const commentList = document.createElement('div');
+  commentList.className = 'comment-list';
+  commentWindow.appendChild(commentList);
+  allComments.forEach((comment) => {
+    const commentItem = document.createElement('div');
+    commentItem.className = 'comment-item';
+    commentItem.innerHTML = `
+      <div class="comment-info">
+        <p class="comment-text">${comment.username}:  ${comment.comment}</p>
+      </div>
```
Comment (Achilles-Dev) [inline modules/add.js:85]: According to the wireframe, comments are supposed to display above the comments form. I would suggest you stick to the design of the wireframe.

**Review by Achilles-Dev — CHANGES_REQUESTED** [review]
Good job!
Everything is working fine. However, there are a few suggestions in the comments section that needs your attention

File: modules/add.js (hunk 1 above)
Comment (Fsher07) [reply modules/add.js]: I fixed it

Comment (Fsher07) [reply modules/add.js]: Yes, you are right. I did my best and after a few code lines, it looks more like wireframe 😄

File: src/style.css [hunk 2]
```diff
@@ -209,16 +209,12 @@ header {
...
-  grid-template-columns: repeat(4, 1fr);
+  margin-top: 0.5rem;
+  grid-template-columns: repeat(5, 1fr);
   row-gap: 5px;
 }
 
 .comment-text {
   display: flex;
   font-size: 10px;
 }
```
Comment (Achilles-Dev) [inline src/style.css:235]: The size of the text is very small. I suggest you increase the size to enable easy reading.

**Review by Achilles-Dev — APPROVED** [review]
Everything looks good but for a small suggestion. 

You can merge now.

Happy coding!

**Pull request merged**